test/testenv/mcutil.go

```diff
@@ -135,9 +136,9 @@ func MCPodReady(ns string, deployment *Deployment) {
 }
 
 // CheckPodNameOnMC Check Standalone Pod configured on MC
-func CheckPodNameOnMC(ns string, deploymentName string, podName string) bool {
+func CheckPodNameOnMC(ns string, mcName string, podName string) bool {
```

Comment:
Nit: Please correct the comment statement here, as this function checks for all peers rather than just standalone